source/connect/authentication-connection.txt

```diff
@@ -13,3 +13,129 @@ Authentication Connection Tab
    :class: singlecol
 
 
+The :guilabel:`Authentication` tab allows you to connect deployments that 
+require authentication. To learn about authentication mechanisms within MongoDB,
+see :driver:`Authentication Mechanisms </go/current/fundamentals/auth/>`
+
+Procedure
+---------
+
+.. procedure::
+   :style: normal
+
+   .. step:: Click :guilabel:`Advanced Connection Options`.
+
+      .. figure:: /images/compass/advanced-connection-options.png
+         :figwidth: 690px
+         :alt: New Advanced Connection Options
+
+   .. step:: Click the :guilabel:`Authentication` tab.
+
+      Select your authentication method from the following options: 
+      
+      - :ref:`Username/Password <username-password>`
+      
+      - :ref:`X.509 <x509>`
+      
+      - :ref:`Kerberos <kerberos-authentication>`
+      
+      - :ref:`LDAP <ldap-authentication>`
+      
+      - :ref:`AWS IAM <aws-iam-authentication>`
+
+      .. _username-password:
+
+      Username/Password 
+      ~~~~~~~~~~~~~~~~~
+
+      Provide the following information:
+
+      - Username
+
+      - Password
+
+      - (Optional) :manual:`Authentication Database </core/security-users/#authentication-database>`
+
+      - Select an Authentication Mechancism:
```

Comment:
Saying "Select an Auth Mechanism" after we've already told them "Provide the following information" before the bullets seems a bit redundant.

I would just make this bullet read "Authentication Mechanism: "

Reply:
Done.